spring-cloud-gcp-samples/spring-cloud-gcp-data-jpa-sample/README.adoc

```diff
@@ -4,30 +4,35 @@ This sample application demonstrates how to use
 https://projects.spring.io/spring-data-jpa/[Spring Data JPA] with the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql[Spring Cloud GCP SQL Starter].
 
+We will create an instance, a database within the instance, populate the database and then query it.
+
 == Setup
 
-* If you don't have a Google Cloud Platform project with billing enabled, create one from the
-https://console.cloud.google.com/[Google Cloud Console].
+1. Follow https://cloud.google.com/sql/docs/mysql/quickstart[these instructions] to set up a Google
+Cloud Project with billing enabled.
 
-* To authenticate, you have two options:
+2. Create a Google Cloud SQL instance from the
+https://console.cloud.google.com/sql/instances[Google Cloud Console Cloud SQL page].
 
-1. Download the https://cloud.google.com/sdk/[Google Cloud SDK] and run the following command:
-+
-`$ gcloud auth application-default login`
+3. Still within the Google Cloud Console SQL page, create a new database in the instance from the
+"Databases" section. **We recommend you to setup a root password** for your instance (in PostgreSQL you have to).
```

Comment:
I have reservations asking users to set a password on their instance.
With the Cloud SQL tunnel + OAuth2 authentication and authorization, the password is actually not necessary and if I had a recommendation, it would actually recommend against setting a password.

Reply:
With PostgreSQL you have to set it, there is no way out. 
Setting passwords is just simply best practice in general. So I would still go with passwords by default. 
However, we can add a footnote, that if users want to try out for development purposes, there is a "passwordless" option for MySQL.